modules/ROOT/pages/teams.adoc

```diff
@@ -0,0 +1,59 @@
+= Manage User Access Using Teams
+
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+The Teams feature enables you to create groups of users and then assign certain permissions.
+Teams have a cascading permissions structure that helps you manage users and permissions with scalability in mind.
+
+If you have the Organization Administrator permission over your root organization, you can achieve the following using the Teams feature:
+
+* Manage your organization from a simple, global view
+* Group users into teams to easily manage permissions and share assets
+* Assign team maintainers to delegate management of team members and child teams
+* Manage user and team permissions across multiple business groups in one place
+
+Starting with the “Everyone at <Your Organization>" team, which contains all users who join your organization either via an invitation or SSO, you can create specialized _child teams_ that have more specific permissions based on the responsibilities of their members.
+The Teams feature enables you to mimic the structure of your organization and seamlessly manage permissions and team membership as your organization grows.
+
+After your child teams are created, you can invite new users directly to the appropriate team using email invitations or using the xref:configure-teams#invite-users-with-teams[Teams feature interface].
+When you add a user to a team, you can also specify whether they should be a _team maintainer_ (a user who can manage team membership) or a _member_ (a user who receives the permissions outlined in the team structure). Outside of teams, you can still xref:user-permissions.adoc#grant-permissions-to-an-individual-user.[assign specific permissions to individual users] if they require permissions that should not extend to an entire team.
+
+Adopting a team structure in your organization also enables you to share assets more easily. In Anypoint Exchange, you can share an asset with an entire team rather than having to select each user individually.
+
+[NOTE]
+To manage teams permissions, you must have the Organization Administrator permission over the business group that contains resources that you want to give access over to a specific user or a team.
+You can modify any permissions for all teams in your organization only if you have the Organization Administrator permission over the whole organization.
+
+== Permissions Structure
+
+When you are planning your organization’s team structure, consider the following:
+
+* Each member of your organization is a member of the “Everyone at <Your Organization> team, or the _root_ team.
+* Your team structure should reflect the structure of your organization so that you can plan your teams and configure permissions according to member needs.
```

Comment:
```suggestion
* Your team structure should reflect the structure of your organization so that you can plan your teams and configure permissions according to member needs. However, you can move teams as the needs of your organization change.
```